label_studio/server.py

```diff
@@ -173,6 +190,13 @@ def validation_error_handler(error):
     logger.error(error)
     return str(error), 500
 
+@app.errorhandler(Exception)
+def handle_error(error):
+    logger.error(error)
+    code = 500
+    if isinstance(error, HTTPException):
+        code = error.code
+    return str(error), code
```

Comment:
Setting all HTTP errors as errors & logging to Bugsnag (as assuming people won't be getting random 404s and there's no real security, so anything that does occur feels like a genuine bug or an attack